src/Symfony/Component/Routing/Loader/XmlFileLoader.php

```diff
@@ -104,33 +104,31 @@ public function supports($resource, string $type = null)
     /**
      * Parses a route and adds it to the RouteCollection.
      *
-     * @param \DOMElement $node     Element to parse that represents a Route
-     * @param string      $filepath Full path of the XML file being processed
+     * @param \DOMElement $node Element to parse that represents a Route
+     * @param string      $path Full path of the XML file being processed
```

Comment:
should not be re-reverted in the follow-up PR for 5.2 (same for all this file: the changes put some lines back in their previous state and are fine as is, to reduce the diff between branches)